scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -158,16 +155,35 @@ package object hdfs {
       }
     }
 
-    private[scio] def hadoopDistCacheCopy(src: Path, target: URI, conf: Configuration): Unit = {
-      logger.debug(s"Will copy ${src.toUri}, to $target")
+    private def getTargetFromOptions() = {
+      if (self.options.getRunner() != classOf[DirectPipelineRunner]) {
+        val dfOptions = self.optionsAs[DataflowPipelineOptions]
+        require(self.optionsAs[DataflowPipelineOptions].getStagingLocation != null,
+          "Staging directory not set - use `--stagingLocation`!")
+        (dfOptions.getStagingLocation, gcsOutputStream _)
+      } else {
+        // should targetDir be specified in options?
```

Comment:
I just mean that the "staging location" here is created with `Files.createTempDirectory` using its default directory - just wondering if we should have this configurable via Options and require it be set. But this is just for testing really so it's probably not an issue to just do this.